test/onnx/test_models_onnxruntime.py

```diff
@@ -41,5 +63,357 @@ def exportTest(self, model, inputs, rtol=1e-2, atol=1e-7, opset_versions=None):
+    )
+
+
+def _get_features(images):
+    s0, s1 = images.shape[-2:]
+    features = [
+        ("0", torch.rand(2, 256, s0 // 4, s1 // 4)),
+        ("1", torch.rand(2, 256, s0 // 8, s1 // 8)),
+        ("2", torch.rand(2, 256, s0 // 16, s1 // 16)),
+        ("3", torch.rand(2, 256, s0 // 32, s1 // 32)),
+        ("4", torch.rand(2, 256, s0 // 64, s1 // 64)),
+    ]
+    features = OrderedDict(features)
+    return features
+
+
+def _init_test_generalized_rcnn_transform():
+    min_size = 100
+    max_size = 200
+    image_mean = [0.485, 0.456, 0.406]
+    image_std = [0.229, 0.224, 0.225]
+    return transform.GeneralizedRCNNTransform(min_size, max_size, image_mean, image_std)
+
+
+def _init_test_rpn():
+    anchor_sizes = ((32,), (64,), (128,), (256,), (512,))
+    aspect_ratios = ((0.5, 1.0, 2.0),) * len(anchor_sizes)
+    rpn_anchor_generator = rpn.AnchorGenerator(anchor_sizes, aspect_ratios)
+    out_channels = 256
+    rpn_head = rpn.RPNHead(
+        out_channels, rpn_anchor_generator.num_anchors_per_location()[0]
+    )
+    rpn_fg_iou_thresh = 0.7
+    rpn_bg_iou_thresh = 0.3
+    rpn_batch_size_per_image = 256
+    rpn_positive_fraction = 0.5
+    rpn_pre_nms_top_n = dict(training=2000, testing=1000)
+    rpn_post_nms_top_n = dict(training=2000, testing=1000)
+    rpn_nms_thresh = 0.7
+    rpn_score_thresh = 0.0
+
+    return rpn.RegionProposalNetwork(
+        rpn_anchor_generator,
+        rpn_head,
+        rpn_fg_iou_thresh,
+        rpn_bg_iou_thresh,
+        rpn_batch_size_per_image,
+        rpn_positive_fraction,
+        rpn_pre_nms_top_n,
+        rpn_post_nms_top_n,
+        rpn_nms_thresh,
+        score_thresh=rpn_score_thresh,
+    )
+
+
+def _init_test_roi_heads_faster_rcnn():
+    out_channels = 256
+    num_classes = 91
+
+    box_fg_iou_thresh = 0.5
+    box_bg_iou_thresh = 0.5
+    box_batch_size_per_image = 512
+    box_positive_fraction = 0.25
+    bbox_reg_weights = None
+    box_score_thresh = 0.05
+    box_nms_thresh = 0.5
+    box_detections_per_img = 100
+
+    box_roi_pool = ops.MultiScaleRoIAlign(
+        featmap_names=["0", "1", "2", "3"], output_size=7, sampling_ratio=2
+    )
+
+    resolution = box_roi_pool.output_size[0]
+    representation_size = 1024
+    box_head = faster_rcnn.TwoMLPHead(
+        out_channels * resolution**2, representation_size
+    )
+
+    representation_size = 1024
+    box_predictor = faster_rcnn.FastRCNNPredictor(representation_size, num_classes)
+
+    return roi_heads.RoIHeads(
+        box_roi_pool,
+        box_head,
+        box_predictor,
+        box_fg_iou_thresh,
+        box_bg_iou_thresh,
+        box_batch_size_per_image,
+        box_positive_fraction,
+        bbox_reg_weights,
+        box_score_thresh,
+        box_nms_thresh,
+        box_detections_per_img,
+    )
+
+
+@parameterized.parameterized_class(
+    ("opset_version", "is_script"),
+    itertools.product([_constants.onnx_default_opset], [True, False]),
```

Comment:
Why are we parameterizing with only one opset version?
Seems unnecessary, since that's the default for the base class anyways.

Reply:
Good catch, getting rid of it. Previously more versions were included, but I think it's unnecessary.